PWGMM/Mult/Tasks/dndeta-mft.cxx

```diff
@@ -425,9 +425,10 @@ struct PseudorapidityDensityMFT {
         if (std::find(ambTrackIds.begin(), ambTrackIds.end(), track.globalIndex()) != ambTrackIds.end()) {
           continue;
         }
-
+        float phi = track.phi();
+        o2::math_utils::bringTo02Pi(phi);
         registry.fill(HIST("Tracks/Centrality/EtaZvtx"), track.eta(), z, c);
-        registry.fill(HIST("Tracks/Centrality/PhiEta"), track.phi(), track.eta(), c);
+        registry.fill(HIST("Tracks/Centrality/PhiEta"), phi, track.eta(), c);
```

Comment:
Why not do it inline: `registry.fill(HIST("Tracks/Centrality/PhiEta"), o2::math_utils::bringTo02Pi(track.phi()), track.eta(), c);`.

Reply:
Did not think about that, will do !